sklearn/decomposition/online_lda.py

```diff
@@ -228,8 +228,11 @@ class LatentDirichletAllocation(BaseEstimator, TransformerMixin):
     Attributes
     ----------
     components_ : array, [n_topics, n_features]
-        Topic word distribution. ``components_[i, j]`` represents word j in
-        topic `i`.
+        Variational parameters for topic word distribution. Since the complete
+        conditional for topic word distribution is a Dirichlet,
+        ``components_[i, j]`` can be viewed as pseudocount that represents the
+        number of times word `j` was assigned to topic `i`.
+        Normalized ``components_`` can be viewed as topic word distribution.
```

Comment:
I would make the last part of the docstring even more explicit:

It can be viewed as distribution over the words for each topic after normalization: `model.components_ / model.components_.sum(axis=1)[:, np.newaxis]`.